### PR TITLE
Added information + Typo fixes

### PR DESCRIPTION
**Pull request by GeorgeDaris**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [X ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/accessibility/wai-aria_basics/index.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ This last point is key — To use a screen reader in the first place, your opera
 
 Next, you need to worry about whether the browsers in question support ARIA features and expose them via their APIs, but also whether screen readers recognize that information and present it to their users in a useful way.
 
-1. Browser support is generally quite good — at the time of writing, [caniuse.com](https://caniuse.com/#feat=wai-aria) stated that global browser support for WAI-ARIA was around 88%.
+1. Browser support is generally quite good — at the time of writing, [caniuse.com](https://caniuse.com/#feat=wai-aria) stated that global browser support for WAI-ARIA was around 88% (98.57% as of August 2022).
```
Comment (Josh-Cena) [inline files/en-us/learn/accessibility/wai-aria_basics/index.md:97]: 98.57% is almost universal (>95%). We don't need to mention the number anymore.

Comment (GeorgeDaris) [reply files/en-us/learn/accessibility/wai-aria_basics/index.md:97]: Makes sense. Should I change it?

Comment (teoli2003) [reply files/en-us/learn/accessibility/wai-aria_basics/index.md:97]: Yes, please.

File: files/en-us/learn/accessibility/wai-aria_basics/index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ The problem here is that visually they work, but screen readers can't make any s
...
 
-- **[Roles](/en-US/docs/Web/Accessibility/ARIA/Roles)** — These define what an element is or does. Many of these are so-called landmark roles, which largely duplicate the semantic value of structural elements, such as `role="navigation"` ({{htmlelement("nav")}}) or `role="complementary"` ({{htmlelement("aside")}}). Some other roles describe different pages structures, such as `role="banner"`, `role="search"`, `role="tablist"`, and `role="tab"`, which are commonly found in UIs.
+- **[Roles](/en-US/docs/Web/Accessibility/ARIA/Roles)** — These define what an element is or does. Many of these are so-called landmark roles, which largely duplicate the semantic value of structural elements, such as `role="navigation"` ({{htmlelement("nav")}}) or `role="complementary"` ({{htmlelement("aside")}}). Some other roles describe different page structures, such as `role="banner"`, `role="search"`, `role="tablist"`, and `role="tab"`, which are commonly found in UIs.
```
Comment (scottaohara) [inline files/en-us/learn/accessibility/wai-aria_basics/index.md:76]: change `tab` to `tabpanel` as `tab` is role that represents a control, which is out of place in this list.  `tabpanel` represents a container element, so it is better suited here.

```suggestion
- **[Roles](/en-US/docs/Web/Accessibility/ARIA/Roles)** — These define what an element is or does. Many of these are so-called landmark roles, which largely duplicate the semantic value of structural elements, such as `role="navigation"` ({{htmlelement("nav")}}) or `role="complementary"` ({{htmlelement("aside")}}). Some other roles describe different page structures, such as `role="banner"`, `role="search"`, `role="tablist"`, and `role="tabpanel"`, which are commonly found in UIs.
```

Comment (GeorgeDaris) [reply files/en-us/learn/accessibility/wai-aria_basics/index.md]: I committed your suggestion. Thanks for the help!

File: files/en-us/learn/accessibility/wai-aria_basics/index.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ This last point is key — To use a screen reader in the first place, your opera
 
 Next, you need to worry about whether the browsers in question support ARIA features and expose them via their APIs, but also whether screen readers recognize that information and present it to their users in a useful way.
 
-1. Browser support is generally quite good — at the time of writing, [caniuse.com](https://caniuse.com/#feat=wai-aria) stated that global browser support for WAI-ARIA was around 88%.
+1. Browser support is almost universal, as stated by [caniuse.com](https://caniuse.com/#feat=wai-aria).
```
Comment (Josh-Cena) [inline files/en-us/learn/accessibility/wai-aria_basics/index.md:97]:
```suggestion
1. Browser support is almost universal, according to [caniuse.com](https://caniuse.com/#feat=wai-aria).
```

Comment (teoli2003) [reply files/en-us/learn/accessibility/wai-aria_basics/index.md:97]: Note that caniuse.com use our data to know which browser support what, so I think we just need to say:
```suggestion
1. Browser support is almost universal.
```

File: files/en-us/learn/accessibility/wai-aria_basics/index.md (hunk 1 above)
Comment (GeorgeDaris) [reply files/en-us/learn/accessibility/wai-aria_basics/index.md]: This should be complete now. I'm still new to GitHub, but I committed  the suggestions from other members and made some changes myself. Do I click on "resolve conversation"?

File: files/en-us/learn/accessibility/wai-aria_basics/index.md [hunk 4]
```diff
@@ -94,7 +94,7 @@ This last point is key — To use a screen reader in the first place, your opera
 
 Next, you need to worry about whether the browsers in question support ARIA features and expose them via their APIs, but also whether screen readers recognize that information and present it to their users in a useful way.
 
-1. Browser support is generally quite good — at the time of writing, [caniuse.com](https://caniuse.com/#feat=wai-aria) stated that global browser support for WAI-ARIA was around 88%.
+1. Browser support is almost universal, according to [caniuse.com](https://caniuse.com/#feat=wai-aria).
```
Comment (teoli2003) [inline files/en-us/learn/accessibility/wai-aria_basics/index.md:97]:
```suggestion
1. Browser support is almost universal).
```

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/learn/accessibility/wai-aria_basics/index.md [hunk 5]
```diff
@@ -94,7 +94,7 @@ This last point is key — To use a screen reader in the first place, your opera
 
 Next, you need to worry about whether the browsers in question support ARIA features and expose them via their APIs, but also whether screen readers recognize that information and present it to their users in a useful way.
 
-1. Browser support is generally quite good — at the time of writing, [caniuse.com](https://caniuse.com/#feat=wai-aria) stated that global browser support for WAI-ARIA was around 88%.
+1. Browser support is almost universal).
```
Comment (teoli2003) [inline files/en-us/learn/accessibility/wai-aria_basics/index.md:97]:
```suggestion
1. Browser support is almost universal.
```

**Pull request merged**